pennylane/devices/default_mixed.py

```diff
@@ -163,23 +163,10 @@ def density_matrix(self, wires):
             array[complex]: complex array of shape ``(2 ** len(wires), 2 ** len(wires))``
             representing the reduced density matrix of the state prior to measurement.
         """
-        # Return the full density matrix if all the wires are given
-        if wires == self.wires:
-            return self.state
+        wires = wires.tolist()
```

Comment:
Is it actually mandatory that `wires` be a list?

Reply:
No you are right `Wires` is also working here